src/redux/categories/categoriesSlice.js

```diff
@@ -0,0 +1,14 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  catList: [], status: 'Under construction',
+};
+
+export const categoriesSlice = createSlice({
+  name: 'categories',
+  initialState,
+  reducers: {},
```

Comment:
- [ ] You have done an excellent job creating the Categories Slice :+1:. However, it is a requirement of the project to include a reducer that checks the status and returns "Under construction". By including this reducer, you can help to ensure that the Categories Slice functions as expected and that the project is completed successfully. 

hint: The initial state should be and empty array and the reducer should look like this:

```js
  reducers: {
    checkStatus: (state, action) => {
      state.categories =
        action.payload === "Under construction"
          ? "Under construction"
          : state.categories
    },
```